src/components/TalentsListPage/SearchPanel/SearchPanel.module.scss

```
@@ -0,0 +1,35 @@
+.search-panel {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap-reverse;
+  margin-top: 80px;
+}
+
+.search-input {
+  color: aliceblue;
```

text in input must be #FFF color according to design
```suggestion
  color: #FFF;
```